services/m3dbnode/config/config.go

```diff
@@ -71,6 +71,14 @@ type Configuration struct {
 	// Write new series backoff between batches of new series insertions.
 	WriteNewSeriesBackoffDuration time.Duration `yaml:"writeNewSeriesBackoffDuration"`
 
+	// Tick per series sleep at the completion of a tick batch, only used
```

Comment:
whats a tick batch, a group of series?

Reply:
Group of series yeah, I can make that more explicit by using group rather than batch.